src/main/java/com/mentalhealthapi/mentalhealthapi/service/DisorderServiceStub.java

```diff
@@ -8,11 +8,11 @@
 
 @Component
 public class DisorderServiceStub implements IDisorderService {
-
     /**
      * Fetch all disorders
      *
      * @return
+     *
```

Comment:
Multiple line comments could have an extra line to better match the examples on medium.com. https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3